packages/contentstack-export/src/config/default.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     types: [
       'stack',
       'assets',
+      'custom-roles',
```

Comment:
keep the custom roles just above the workflows, is there any other module dependent on custom roles ?